test/integration/rules/role-img-alt/role-img-alt.html

```diff
@@ -0,0 +1,13 @@
+<div id="match">Bananas</div>
+<div id="hidden-match" style="display:none">Banana bombs</div>
```

Comment:
Did you mean to put `role="img"` here?

Reply:
No, that's purely for testing `aria-labelledby`, it's a pattern used in other `alt` tests.